src/theme/table/error.js

```diff
@@ -4,13 +4,18 @@ import styles from '../box/emptyState.css';
 
 const Error = ({ data, isLoading }) => {
   const notFoundReg = /not\sfound/gi;
-  if (isLoading || !data || (typeof data === 'string' ? notFoundReg.test(data) : notFoundReg.test(data.message))) {
+  if (
+    isLoading ||
+    !data ||
+    data.response?.status === 404 ||
```

Comment:
response status 404 is going to change to 200 please remove this logic

Reply:
@soroushm I wonder if it's best to wait for it to be updated before we implement the change so we can test extensively